src/scrub.S

```diff
@@ -11,9 +11,51 @@
 .option push
 .option norelax
 
-.type _metal_memory_scrub, @function
-_metal_memory_scrub:
-    /* Zero out memory */
+/* Function to zero-scrub specified memory */
```

Comment:
I'd love to see the interface to this function documented in comments here.

Reply:
Updated comments with function input parameter details.